data processing/csv_to_json.py

```diff
@@ -0,0 +1,35 @@
+import csv, json, datetime
+
+# Specify the directory path
+csv_file_name = r'C:\Users\camil\Robotics\JoinedData\PNCMP2024-MatchExport-Q100.csv' #TODO:check if file name need changing later
+json_file_name = r'C:\Users\camil\Robotics\JoinedData\DCMPQualMatches.json'
```

Comment:
Why are these variables only referenced once?

Reply:
Why is there still a "to do"?

Reply:
because there shouldn't be one ╰(*°▽°*)╯